src/ocm/components/clusterConfiguration/networkConfiguration/AdvancedNetworkFields.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import {
+  Alert,
+  AlertVariant,
+  FormGroup,
+  StackItem,
+  TextInputTypes,
+  Grid,
+} from '@patternfly/react-core';
+import { Address6 } from 'ip-address';
+import { FieldArray, useFormikContext } from 'formik';
+import { NetworkConfigurationValues } from '../../../../common/types';
+import { useFeature } from '../../../../common/features';
+import { InputField } from '../../../../common/components/ui';
+import { PREFIX_MAX_RESTRICTION } from '../../../../common/config/constants';
+import { NetworkTypeControlGroup } from '../../../../common/components/clusterWizard/networkingSteps/NetworkTypeControlGroup';
+
+type AdvancedNetworkFieldsProps = {
+  isSDNSelectable: boolean;
+};
+
+const AdvancedNetworkFields: React.FC<AdvancedNetworkFieldsProps> = ({ isSDNSelectable }) => {
+  const { setFieldValue, values, errors } = useFormikContext<NetworkConfigurationValues>();
+
+  const isNetworkTypeSelectionEnabled = useFeature(
+    'ASSISTED_INSTALLER_NETWORK_TYPE_SELECTION_FEATURE',
+  );
+
+  const clusterNetworkCidrPrefix = (index: number) =>
+    parseInt(
+      ((values.clusterNetworks && values.clusterNetworks[index].cidr) || '').split('/')[1],
+    ) || 1;
+
+  const formatClusterNetworkHostPrefix = (
+    e: React.ChangeEvent<HTMLInputElement>,
+    index: number,
+  ) => {
+    if (isNaN(parseInt(e.target.value))) {
+      setFieldValue(`clusterNetworks.${index}.hostPrefix`, clusterNetworkCidrPrefix(index));
+    }
+  };
+
+  const isSubnetIPv6 = (index: number) =>
+    Address6.isValid((values.clusterNetworks && values.clusterNetworks[index].cidr) || '');
```

Comment:
This won't detect IPv6 properly, it should look at formik field hostSubnet

Reply:
I'm confused. There is no such field `hostSubnet` in the form. This is used for the cluster networks subnet+host-prefix pairs to decide the helper texts and min/max values for for each pair. Why would this not detect IPv6 properly?